### ISSUE TITLE
Undefined index: REQUEST_URI after update contacts 0.4.0.0 on oC 8.1

### ISSUE THREAD

**Issue by RealRancor** [owncloud/contacts#972]
Not using the issue template as this is very likely just the same as in the calendar app https://github.com/owncloud/calendar/issues/714 which got fixed by https://github.com/owncloud/calendar/commit/3e4a1e1a241479c2c3c5be60379c520c421ce8c0

This is happening on contacts 0.4.0.0 with oC 8.1 when cron runs (from cli) where REQUEST_URI is not set.

**Comment by tony5** [comment]
I have the same issue or so it seems.
 Undefined index: REQUEST_URI at /var/www/cloud/apps/contacts/appinfo/app.php#35

**Comment by raimund-schluessler** [comment]
Same here.

**Comment by RealRancor** [comment]
Fix is already in https://github.com/owncloud/contacts/issues/973 :-)

**Comment by raimund-schluessler** [comment]
Thanks, I will test it this evening.

**Comment by DeepDiver1975** [comment]
fix has been merged and is also backported to 8.1

**Comment by enoch85** [comment]
@RealRancor Sorry to say, not working:

```
Error  PHP  Undefined index: REQUEST_URI at /var/www/owncloud/apps/contacts/appinfo/app.php#35  2015-08-11T19:15:01+01:00  
Error  PHP  Undefined index: REQUEST_URI at /var/www/owncloud/apps/contacts/appinfo/app.php#35  2015-08-11T19:00:03+01:00  
Error PHP Undefined index: REQUEST_URI at /var/www/owncloud/apps/contacts/appinfo/app.php#35 2015-08-11T18:45:01+01:00 
Error PHP Undefined index: REQUEST_URI at /var/www/owncloud/apps/contacts/appinfo/app.php#35 2015-08-11T18:30:02+01:00 
Error PHP Undefined index: REQUEST_URI at /var/www/owncloud/apps/contacts/appinfo/app.php#35 2015-08-11T18:15:01+01:00 
Error PHP Undefined index: REQUEST_URI at /var/www/owncloud/apps/contacts/appinfo/app.php#35 2015-08-11T18:00:03+01:00 
Error PHP Undefined index: REQUEST_URI at /var/www/owncloud/apps/contacts/appinfo/app.php#35 2015-08-11T17:45:03+01:00 
Error PHP Undefined index: REQUEST_URI at /var/www/owncloud/apps/contacts/appinfo/app.php#35 2015-08-11T17:30:03+01:00 
Error PHP Undefined index: REQUEST_URI at /var/www/owncloud/apps/contacts/appinfo/app.php#35 2015-08-11T17:15:02+01:00 
```

On 8.1.1 stable.

@DeepDiver1975 Wasn't this backported?

**Comment by RealRancor** [comment]
Contacts is not shipped anymore with oC core so an update to 8.1.1 won't fix this. This needs an new release of the contact app which includes the already backported fix.

**Comment by enoch85** [comment]
@RealRancor Ok, I see.
